src/foam/dao/DAO.js

```diff
@@ -83,6 +87,7 @@ foam.INTERFACE({
       async: true,
       type: 'FObject',
       swiftThrows: true,
+      documentation: 'Retrieves a single object from the DAO, whose id is equal to the parameter id. If the object is found, the promise resolves with the object. If the object is not found, it rejects with a foam.dao.ObjectNotFoundException.',
```

Comment:
I don't think this is true. If not found it just returns null rather than throw an Exception. Also, you can specify the id or a predicate.

Reply:
Ok, who should change this in the word documentation?

Reply:
Replaced by:
Retrieves a single object from the DAO, whose id is equal to the parameter id or if the given predicate evaluates to true.
If the object is found, the promise resolves with the object.
If the object is not found, it returns null.